libcontainer/cgroups/fscommon/rdma.go

```diff
@@ -17,14 +17,12 @@ import (
 func parseRdmaKV(raw string, entry *cgroups.RdmaEntry) error {
 	var value uint32
 
-	parts := strings.SplitN(raw, "=", 3)
+	k, v, ok := strings.Cut(raw, "=")
 
-	if len(parts) != 2 {
+	if !ok {
 		return errors.New("Unable to parse RDMA entry")
```

Comment:
Some of these could be improved as well; in this case we know it's missing a `=`.  Wondering if we must also check for _empty_ values after; looks like the `if/else`  below would fall through to trying to parse empty values as integer,  which may produce an obscure error (`strconv.ParseUint: parsing "": invalid syntax`).


We could check here if it's empty and return that no value is specified;

```go
if !ok || v == "" {
	return errors.New("Unable to parse RDMA entry: no value specified")
}
```